### DISCUSSION TITLE
Not able to run zed in kali-linux wsl2

### DISCUSSION THREAD

**Discussion by Aamirusmani1552** [zed-industries/zed#17569] [Feature Requests]
After installing zed. It says that we have to run command `zed` to run it. But after I run the command, nothing happens. I read some other issues that we have to install `vulkan` and other things. I have done all of that. But still facing same issue. I tried opening current folder like `zed .`, still nothing. 

Need to know how can i solve this. Or Am i doing something wrong. Please help

**Comment by HappiePlant** [comment]
Maybe related to this: https://github.com/zed-industries/zed/issues/17129
Maybe try: `DRI_PRIME=1 zed .`

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
